src/modules/Dashboard/api/api.ts

```diff
@@ -984,7 +984,7 @@ export const deleteAppletAssignmentsApi = (
   });
 
 export const enableIntegrationApi = (integrations: Integration[], signal?: AbortSignal) =>
-  authApiClient.post('/integrations/', integrations, {
+  authApiClient.post('/integrations', integrations, {
     signal,
   });
```

Comment:
this endpoint doesn't exist anymore, it was replaced by `saveIntegrationToApi`

Reply:
The one that saves the integration is the one you added as `saveIntegrationToApi`, which takes type, config, etc as parameters. This is the dummy endpoint which didn't include any additional server information.